pinot-core/src/main/java/org/apache/pinot/core/realtime/converter/RealtimeSegmentConverter.java

```diff
@@ -156,20 +156,8 @@ public void build(@Nullable SegmentVersion segmentVersion, ServerMetrics serverM
   @VisibleForTesting
   public Schema getUpdatedSchema(Schema original) {
```

Comment:
We need to remove the time transform here because the time has already been transformed.
Please also update the javadoc

Reply:
I believe we don't need to change the schema because:
1) The RecordTransformer is a pass through
2) The RecordReader is just looping over physical column names so it only has outgoing

We shouldn't need to remove incoming, because that implies that we need to remove transform functions from all field specs.